std/deps.ln

```diff
@@ -1,34 +1,331 @@
-from @std/app import start, print, exit
+from @std/app import start, print, exit, eprint
 from @std/cmd import exec
 
 /**
- * @std/deps - The entrypoint to install dependencies for an alan program
+ * @std/deps - The entrypoint to install dependencies for an alan program 
+ * and define permissions over libraries
  */
 
+type Dependency {
+  url: string,
+  block: Array<string>,
+  fullBlock: Array<string>,
+  group: string,
+  name: string,
+  rev: string,
+  remote: string,
+}
+
+type RawPackage {
+  std: Array<string>,
+  dependencies: Array<Dependency>,
+  block: Array<string>,
+  fullBlock: Array<string>,
+}
+
+type PackageDependency {
+  pkg: RawPackage,
+  dep: Dependency,
+}
+
+// Exporting type as alias since could not export the type and used it in this file at the same time
```

Comment:
That's a weird bug we should tackle. I'm sure it's because we do exports last so the inner functions wouldn't have seen the type.

Does `type Package { ... }` and then `export Package` after it not work?